### DISCUSSION TITLE
Console/system "explore" pages

### DISCUSSION THREAD

**Discussion by wescopeland** [RetroAchievements/RAWeb#2467] [Ideas]
Right now, we currently have system pages like:
https://retroachievements.org/system/4/games

However, these aren't really landing pages. They're just long lists of stuff.

Especially given the upcoming Gamecube release, I think it would be interesting to have dedicated landing pages for each system. The routes for these pages would be something like https://retroachievements.org/system/4. These landing pages would be more heavily targeted towards newer users, containing fun content like:

* Trending games by play count for the week/month
* Community favorites / Most popular games overall
* Most beaten/mastered games
* Least beaten/mastered games
* % of official retail game library covered by achievements
* Top players for the system
* Most recently-released sets for the system
* Historical fun facts (RA rollout date, iconic games, blurbs about the system's impact on gaming itself)
* Game recommendations based on the user's existing play history
* "Play This Set" recommendations/blurbs from RANews
* Featured achievements and challenges
* Random game button which picks a random set from the system and shows it to the user

A lot of this content could be queried/cached automatically, though some of it would need to be handcrafted for each system.

**Comment by suspect15** [comment]
Here are some other ideas that might make for some interesting info/stats:

- Rarest achievement (achievement unlocks/set players, probably should be > 0)
- Number of unique console players (not sure if this can be calculated since many users will play multiple games on the console)
- Highest number of concurrent console players (not sure if this is tracked)
- Total number of achievements on console
- Total number of unlocked achievements on console
- Total number of leaderboards on console
- Total number of leaderboard entries on console
- Number of games with sets
- Number of games without sets
- Number of claimed sets
- Most dropped set (number of different devs who have dropped it)
- Number of developers who've authored achievements on sets
- Number of total console tickets (probably just open and resolved)
- Set with highest number of players
- Set with lowest number of players

**Comment by Blazekickn** [comment]
Would love to see the top 25 mastery/beat leaderboards for each console, similar to the ones in RANews

**Comment by wescopeland** [comment]
Nintendo has been building out an online Famicom museum which may serve as design inspiration for a console landing page: https://www.nintendo.com/jp/famicom/history/index.html

It's pretty information-dense like most Japanese web design tends to be. They also have these game highlights pages that are pretty cool: https://www.nintendo.com/jp/famicom/software/index.html
